CODEOWNERS

```diff
@@ -1,16 +1,16 @@
 # Aaron should see all code changes (things not in /pages)
-# Josh should see ALL PRs
-* @niartenyaw @jpe442
+# taking off from ALL @jpe442
+* @niartenyaw
 
 # Josh take over all content (remove Aaron)
-/pages/ @jpe442
+# /pages/ @jpe442
```

Comment:
There still needs to be this line with no one to take me off

Reply:
You still get all, I left yours open on line 3, no??

Reply:
yeah, but i don't want to get things in `/pages`

Reply:
What happens if I leave /pages blank?? Will it still cancel it out for you by order of precedence but not be able to assign anyone??